psydac/polar/dense.py

```diff
@@ -320,6 +320,9 @@ def domain( self ):
     def codomain( self ):
         return self._codomain
 
+    def transpose(self, conjugate=False):
+        return NotImplementedError()
```

Comment:
You should `raise` the exception instead of returning it:
```suggestion
    def transpose(self, conjugate=False):
        raise NotImplementedError()
```